### DISCUSSION TITLE
課題13についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#45] [Q&A]
common13の2つ目のfor文の使い方がわかりませんでした。どこで使用するのでしょうか？

**Comment by nsc-kiwamu** [comment]
2つ目のfor文はどれのことでしょうか？

**Reply by TakedaAyano** [reply]
この部分のコードになります。

export function makeRandomList(size: number, min:number, max:number): number[] {
    const retList:number[] = []

    for (let i = 0; i < size; i++) {
        retList.push(makeRandom(min, max))
    }

    return retList
}
